index.html

```diff
@@ -0,0 +1,243 @@
+                    </div>
+                </div>            
+
+                <div class="grid-item">
+                    <div class="profesional"><h2 class="prof-art">Profesional Art Printing Data</h2></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    <div class="buttons">
+                        <div class="but-html">HTML</div>
+                        <div class="but-boot">Bootstrap</div>
+                        <div class="but-ruby">Ruby</div>
+                    </div>
+                    <div class="orange-button1">
+                        <div class="see-project1">See project</div>
+                    </div>
+                </div>           
+                
+                <div class="grid-item">
+                    <div class="profesional"><h2 class="prof-art">Profesional Art Printing Data</h2></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    <div class="buttons">
+                        <div class="but-html">HTML</div>
+                        <div class="but-boot">Bootstrap</div>
+                        <div class="but-ruby">Ruby</div>
+                    </div>
+                    <div class="orange-button1">
+                        <div class="see-project1">See project</div>
+                    </div>
+                </div>    
+                                
+                <div class="grid-item">
+                    <div class="profesional"><h2 class="prof-art">Profesional Art Printing Data</h2></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    <div class="buttons">
+                        <div class="but-html">HTML</div>
+                        <div class="but-boot">Bootstrap</div>
+                        <div class="but-ruby">Ruby</div>
+                    </div>
+                    <div class="orange-button1">
+                        <div class="see-project1">See project</div>
+                    </div>
+                </div>
+
+                <div class="grid-item">
+                    <div class="profesional"><h2 class="prof-art">Profesional Art Printing Datas</h2></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    <div class="buttons">
+                        <div class="but-html">HTML</div>
+                        <div class="but-boot">Bootstrap</div>
+                        <div class="but-ruby">Ruby</div>
+                    </div>
```

Comment:
- [ ] Please 🙏🏻  use the `<ul>` tag over `div` to enhance the professionalism and semantic correctness of your code, use the `<ul>` tag as a container and place each social media icon link within an individual **li** tag. This approach not only improves the structure of your *HTML* but also benefits **visually-impaired** individuals, ensuring a better user experience. Avoid using the `<div>` tag solely for decorative purposes, as it lacks semantic meaning.

   - Once you make the necessary changes, remember to add appropriate styles to your CSS file. Implementing 
     **Flexbox** can help you properly align the items within the **ul** container. Utilizing `Flexbox` provides a flexible 
     and responsive layout, ensuring consistent presentation across different screen sizes and devices.

   - It is crucial to always close your tags with the correct tag name. This practice becomes even more significant as your code scales or when you add more features in the future. By diligently closing your tags, you avoid 
     potential errors that may arise due to small oversights. Maintaining proper tag closure ensures code readability, makes debugging easier, and helps maintain the integrity of your markup.

- [ ] Kindly stick with the *Figma* design as closely as possible, it seems like your font family is not matching in your work using the `Inter` font family. Please have a look at the below comparison table to understand the problem in a better way through visualization.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-06 13-14-41](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/aa9935b2-4d54-4722-bbce-0ce9913e828b)


</td>

<td>

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/5b62ae2f-fba1-426f-b16c-31830c112d6a)


</td>
</tr>
</table>


#### Requirement

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/641d0c3f-f77f-4683-a5d1-b0c8780bdbe6)